build.fsx

```diff
@@ -89,14 +89,14 @@ let watchDocs _ =
                                                 WorkingDir = workingDir
                                                 TimeOut =  TimeSpan.FromDays 1. } ) // Really big timeout as we use a watcher
 
-    runDotnetNoTimeout "docs" "fable node-run ../node_modules/rollup/bin/rollup --port free -- -c rollup-config.js -w"
+    runDotnetNoTimeout "docs" "fable node-run ../node_modules/webpack-dev-server/bin/webpack-dev-server.js --port free -- --config webpack.config.js"
```

Comment:
There's a Fable command for this ;) You can just run `dotnet fable webpack-dev-server --port free` (I think config is not needed, I've to check)

Reply:
So, this command is new no ?
Because, we was needing the `node-run` in the past.

But smaller is better, will update it :)